aptos-move/parallel-executor/src/proptest_types/tests.rs

```diff
@@ -152,10 +162,53 @@ fn dynamic_read_writes_contended() {
     .current();
 
     assert!(run_transactions(
-        universe,
+        &universe,
+        transaction_gen,
+        vec![],
+        vec![],
+        100,
+        (false, false),
+    ));
+}
+
+#[test]
+fn module_publishing_fallback() {
```

Comment:
General remark (no action for this PR): I'm not sure whether combinatorial testing is good enough here. It's hard to see whether you actually cover the the particular scenario. I would feel saver when some scenarios would be explicitly constructed. Also there must be some way for concurrency testing in Rust to instrument thread scheduling.

Reply:
Yeah, this test is a bit weird, the booleans indicate if read or written paths are interpreted as module paths, respectively. When both are true, with overwhelming probability (I computed, more than 1-1/2^3000), it has an intersection, thus basically guaranteed that it should return the intersection error.

I'll note to self to follow up with some PRs to add smaller, explicit tests for this and as Zekun suggested, a bigger, non-fake test with real module paths.